src/styles.scss

```diff
@@ -1,5 +1,7 @@
 /* You can add global styles to this file, and also import other style files */
 @import '~@angular/material/prebuilt-themes/deeppurple-amber.css';
+@import '../node_modules/font-awesome/scss/font-awesome.scss';
```

Comment:
can we use here the same import as for `@angular/material` starting from `~`? E.g. 
```
@import '~font-awesome/scss/font-awesome.scss`
```

Reply:
can we use instead of font-awesome material icon, that already included into angular-material? That will reduce amount of dependencies and bundle size.

Reply:
Yep, it works.
Concidering material icons: I'd like to use fa to gain more experience in working with it including extracting icons as a component